packages/web/src/dev/ValidatorsListRow.tsx

```diff
@@ -85,61 +48,33 @@ class ValidatorsListRow extends React.PureComponent<Props & I18nProps, State> {
       document.removeEventListener('click', onDocumentClick, false)
   }
 
-  componentDidMount() {
-    this.setState({ isPinned: this.isPinned() })
-  }
-
   componentWillUnmount() {
     this.removeDocumentListener()
   }
 
-  isPinned(toggle?: boolean) {
-    const { address } = this.props.group
-    let list = (localStorage.getItem(localStoragePinnedKey) || '').split(',') || []
-    let isPinned = list.includes(address)
-    if (toggle) {
-      if (!isPinned) {
-        list.push(address)
-      } else {
-        list = list.filter((_) => _ !== address)
-      }
-      isPinned = !isPinned
-      localStorage.setItem(localStoragePinnedKey, list.join(','))
-      this.props.onPinned()
-    }
-    return isPinned
-  }
-
-  stopPropagation = (event) => {
-    event.stopPropagation()
-  }
   toggleTooltip = (event) => {
     event.stopPropagation()
     this.setState({ tooltip: !this.state.tooltip })
   }
-  togglePinned: any = (event) => {
-    event.stopPropagation()
-    const is = this.isPinned(true)
-    this.setState({ isPinned: is })
-  }
 
   render() {
     const { group, expanded } = this.props
-    const { tooltip, isPinned } = this.state
+    const { tooltip } = this.state
 
     return (
       <div style={tooltip ? { zIndex: 2 } : {}}>
         <View style={[styles.tableRow, styles.tableRowCont, tooltip ? { zIndex: 3 } : {}]}>
-          <View
-            style={[styles.tableCell, styles.pinContainer, styles.sizeXXS]}
-            onClick={this.togglePinned}
-          >
-            <View style={[styles.pin, isPinned ? styles.pinned : {}]} />
-          </View>
           <View style={[styles.tableCell, styles.tableCellTitle]}>
-            <Text style={[styles.defaultText, styles.tableCell, styles.tableCellTitleArrow]}>
+            <Text
+              style={[
+                styles.defaultText,
+                styles.tableCell,
+                styles.tableCellTitleArrow,
+                ...(expanded ? [styles.tableCellTitleArrowExpanded] : []),
```

Comment:
if im reading this right i think `expanded && styles.tableCellTitleArrowExpanded` is all you need here. Saves on making arrays

Reply:
Oh yes, much better!